supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc

```diff
@@ -445,13 +445,25 @@ Always spell out the full product name of the host, and do not capitalize the te
 [discrete]
 [[red-hat-openshift-container-storage]]
 ==== Red Hat OpenShift Container Storage (noun)
-*Description*: Red Hat software-defined storage for containers that helps to develop and deploy applications quickly and efficiently across cloud platforms.
+*Description*: Red Hat software-defined storage for containers that helps to develop and deploy applications quickly and efficiently across cloud platforms. In 2021, Red Hat OpenShift Container Storage was rebranded as Red Hat OpenShift Data Foundation.
 
 *Use it*: yes
 
 *Incorrect forms*: OCS
 
-*See also*:
+*See also*: xref:red-hat-openshift-data-foundation[Red Hat OpenShift Data Foundation]
+
+// Added entry for ODF and updated OCS entry
+[discrete]
+[[red-hat-openshift-data-foundation]]
+==== Red Hat OpenShift Data Foundation (noun)
```

Comment:
This is missing the "yes" icon ahead of the term name. I know you just copy/pasted it from the above entry (which was incorrect), but would you mind adding in the icon now?

It should look like this:
```
==== image:images/yes.png[yes] Red Hat OpenShift Data Foundation (noun)
```

You could also fix the above entry to use the "no" one if you wouldn't mind:

```
==== image:images/no.png[no] Red Hat OpenShift Container Storage (noun)
```

Reply:
Will do!